v3/plugins/fwcd/vscode-kotlin/0.2.18/meta.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+publisher: fwcd
+name: vscode-kotlin
+version: 0.2.18
+type: VS Code extension
+displayName: Kotlin
+title: Kotlin tooling for VS Code
+description: This plug-in provides support for Kotlin development.
+icon: https://raw.githubusercontent.com/JetBrains/logos/master/web/kotlin/kotlin.svg
+repository: https://github.com/fwcd/vscode-kotlin/
+category: Language
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11-7bd8c8c"
+      name: vscode-quarkus-kotlin
```

Comment:
If quarkus isn't part of this plugin I'd say just name this `vscode-kotlin`

Reply:
I can confirm there's no result for "quarkus" search within the kotlin extension source repo. So I don't think there's a direct/indirect dependency on it.

Reply:
updated thanks, the quarkus part is only for the devfile I pushed also to `che-devfile-registry` using this plugin together with quarkus one